.github/workflows/bindings.yml

```diff
@@ -0,0 +1,102 @@
+name: Bindings
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - '**.md'
+      - 'demo/**'
+  pull_request:
+    branches: [ master ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  android:
+    runs-on: ${{ matrix.os }}
```

Comment:
Do we really need a matrix here? If there any chances that we will add more platforms for builds in the future?